config/environments/production.rb

```diff
@@ -123,4 +123,8 @@
   # subsequently, this will cause the SSL certs registered for our private DNS address to fail.
   config.action_controller.default_url_options = { host: primary_domain }
   config.action_controller.asset_host = primary_domain
+
+  Raven.configure do |config|
```

Comment:
What do you think about putting this block and https://github.com/ifmeorg/ifme/pull/1103/files#diff-58da95d5e273d16d883532efa37ca499 together. 
So we would have:
```
Raven.configure do |config|
  config.dsn = ENV['SENTRY_DSN']
  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
end
```

Reply:
Ooh, that would technically deduce the need for having that initializer file. Good idea!